src/pages/components/overview/feature-flags.mdx

```diff
@@ -0,0 +1,81 @@
+| Menu buttons (Overflow menu) | No design change                                                                                               | <Link href={"https://react.carbondesignsystem.com/?path=/docs/experimental-feature-flags-overflowmenu--overview"} renderIcon={() => <Launch aria-label="Launch" />}>React</Link>             |
+| Modal                        | No design change                                                                                               | <Link href={"https://react.carbondesignsystem.com/?path=/docs/experimental-feature-flags-modal--overview"} renderIcon={() => <Launch aria-label="Launch" />}>React</Link>                    |
+| Notification                 | No design change                                                                                               | <Link href={"https://react.carbondesignsystem.com/?path=/docs/experimental-feature-flags-notifications-actionable--overview"} renderIcon={() => <Launch aria-label="Launch" />}>React</Link> |
+| Structured list              | <Link href={"https://carbondesignsystem.com/components/structured-list/usage/#feature-flags"}>Usage tab</Link> | <Link href={"https://react.carbondesignsystem.com/?path=/docs/experimental-feature-flags-structuredlist--overview"} renderIcon={() => <Launch aria-label="Launch" />}>React</Link>           |
+| Tile                         | <Link href={"https://carbondesignsystem.com/components/tile/usage/#feature-flags"}>Usage tab</Link>            | <Link href={"https://react.carbondesignsystem.com/?path=/docs/experimental-feature-flags-tile--overview"} renderIcon={() => <Launch aria-label="Launch" />}>React</Link>                     |
+| Tree view                    | No design change                                                                                               | <Link href={"https://react.carbondesignsystem.com/?path=/docs/experimental-feature-flags-treeview--overview"} renderIcon={() => <Launch aria-label="Launch" />}>React</Link>                 |
+
+## How to implement
+
+### Code
+
+In React, for more feature flag information specific to developers, refer to the
+[@carbon/react](https://react.carbondesignsystem.com/?path=/docs/experimental-feature-flags--overview)
+framework.
+
+### Design
+
+Inside a design file, click on the component with a feature flag (A). In the
+properties panel, locate the "**Feature flag**" boolean property (B). By
+default, this feature is turned off, but you can toggle it on to enable the
+feature flag.
+
+<Row>
+<Column colLg={8}>
+
+![Feature flag implementation in Figma](images/feature-flag-implementation-1.png)
+
+</Column>
+</Row>
+
+Additionally, when viewing more details in the "**Component documentation**" (C)
```

Comment:
This section can be rewritten to something like this to make it more clear of how to navigate to this information. This was already in the Figma file, I just think it didn't get updated here: 

> Additionally, when clicking on the "**components details**" (C) in the properties panel, a description of what a feature flag is and a link to the components usage guidance is provided for more information in the "**Component documentation**" window (D).

Reply:
Oh okay, updated now. Thank you!